src/ArticlePage.jsx

```diff
@@ -1,15 +1,16 @@
 import { useLocation } from "react-router-dom";
-import { getArticleById } from "../API";
+import { getArticleById, getCommentsByArticleId } from "../API";
 import { useEffect, useState } from "react";
 import "./cssFiles/ArticlePage.css";
 
 export default function ArticlePage() {
   const location = useLocation();
   const { id } = location.state;
```

Comment:
It shouldn't be necessary to be using `useLocation` to get the parameter from the URL, this is slightly more work than using `useParams` [check notes for useParams](l2c.northcoders.com/courses/sd-notes/front-end/#sectionId=react-router,step=parametric-endpoints-with-useparams)